src/database/common.c

```diff
@@ -261,6 +256,11 @@ void db_init(void)
 		}
 	}
 
+		// Explicitly set permissions to 0664
+		// 664 =            u+w       u+r       g+w       g+r       o+r
+		const mode_t mode = S_IWUSR | S_IRUSR | S_IWGRP | S_IRGRP | S_IROTH;
+		chmod_file(FTLfiles.FTL_db, mode);
+
```

Comment:
Sorry, one last but, then we are ready to merge: Indentation is broken here.
![Screenshot from 2021-09-23 20-51-52](https://user-images.githubusercontent.com/16748619/134566639-fa2f710f-25bf-44e6-b0c7-6f2ac5b7dad2.png)
```suggestion
	// Explicitly set permissions to 0664
	// 664 =            u+w       u+r       g+w       g+r       o+r
	const mode_t mode = S_IWUSR | S_IRUSR | S_IWGRP | S_IRGRP | S_IROTH;
	chmod_file(FTLfiles.FTL_db, mode);

```